src/pages/signIn/type.tsx

```diff
@@ -0,0 +1,17 @@
+import { SubmitHandler, FieldValues } from 'react-hook-form';
+
+export type AuthData = {
+  username: string;
+  password: string;
+};
+export type FormValues = AuthData & {
+  confirmpassword: string;
```

Comment:
название должно быть через camelCase
```suggestion 
confirmPassword
```